src/app/admin/admin.component.ts

```diff
@@ -38,11 +38,11 @@ export class AdminComponent implements OnInit, OnDestroy {
     private messagesService: MessagesService
   ) {}
 
-  ngOnInit() {
+  async ngOnInit() {
     // hide CLARK navbar
     this.navbarService.hide();
 
-    if (this.messagesService.getStatus()) {
+    if (!!(await this.messagesService.getDowntime()).message) {
```

Comment:
Is this why the banner says lorem ipsum right now, as seen in the demo video?

Reply:
If i understand your question right, no this helps display the banner in the admin dashboard. The code that enables the banner is in `app.component.ts`

Reply:
Ooo right.